service/src/main/java/tds/assessment/repositories/impl/AssessmentQueryRepositoryImpl.java

```diff
@@ -147,6 +149,51 @@ public AssessmentQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTempla
     }
 
     @Override
+    public List<AssessmentInfo> findAssessmentInfoByKeys(final String clientName, final String... assessmentKeys) {
+        final SqlParameterSource parameters = new MapSqlParameterSource("keys", Arrays.asList(assessmentKeys))
+            .addValue("clientName", clientName);
+
+        final String SQL =
+            "SELECT \n" +
+                "   a._key AS assessmentKey, \n" +
+                "   a.testid AS assessmentId, \n" +
+                "   tp.subjectname AS subject, \n" +
+                "   tp.label AS assessmentLabel, \n" +
+                "   GROUP_CONCAT(DISTINCT tool.code) languages, \n" +
+                "   GROUP_CONCAT(DISTINCT g.grade) grades, \n" +
+                "   tp.maxopportunities AS maxAttempts \n" +
```

Comment:
Good point. I don't believe so, not in practice anyway. Theoretically its possible to insert any value into that VARCHAR column ("kin,dergar,den"), but I do not thing this is something to  be seriously concerned about.